libs/paths/Ppath.ml

```diff
@@ -205,7 +205,7 @@ let remove_prefix root path =
   let path = Fpath.to_dir_path path in
   (* now we can call this function to remove the root prefix from path *)
   match Fpath.rem_prefix root path with
-  | None -> None
+  | None -> if Fpath.equal root path then Some (Fpath.v ".") else None
```

Comment:
would be good to add this example in the doc in Ppath.mli and maybe add a test showing this behavior in Unit_ppath.ml

Reply:
Tests and examples are added.